tests/flows/test_fpga_xml_constraints.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+import pytest
+
+import siliconcompiler
+
+from siliconcompiler.targets import fpgaflow_demo
+
+
+@pytest.mark.eda
+@pytest.mark.quick
+def test_fpga_xml_constraints(scroot,
+                              arch_name='example_arch_X014Y014',
+                              benchmark_name='adder',
+                              top_module='adder'):
+
+    chip = siliconcompiler.Chip(f'{top_module}')
+
+    chip.set('fpga', 'partname', arch_name)
+
+    flow_root = os.path.join(scroot, 'examples', 'fpga_flow')
+
+    # 1. Defining the project
+    # 2. Define source files
+    v_src = os.path.join(flow_root, 'designs', benchmark_name, f'{benchmark_name}.v')
+    chip.input(v_src)
+
+    # 3. Set placement constraints
+    xml_constraints = os.path.join(flow_root,
+                                   'designs',
+                                   benchmark_name,
+                                   'constraints',
+                                   f'pin_constraints_{arch_name}.xml')
```

Comment:
since this xml file is only used in the tests, please move it to the tests folder `/tests/flows/data/`

Reply:
Done